prow/config.yaml

```diff
@@ -139,10 +139,9 @@ presubmits:
     trigger: "@k8s-bot (verify )?test this"
 
   - name: pull-kubernetes-e2e-gce
-    always_run: true
-    context: Jenkins GCE e2e
-    rerun_command: "@k8s-bot cvm gce e2e test this"
-    trigger: "@k8s-bot (cvm )?(gce )?(e2e )?test this"
+    context: pull-kubernetes-e2e-gce
```

Comment:
Sure, but the commit message and PR title say "make etcd2 test non-blocking", not "make etcd2 test non-blocking and also rename all the non-blocking job contexts to match their job name".